### ISSUE TITLE
Package structure 

### ISSUE THREAD

**Issue by sai-prasanna** [danijar/dreamerv3#132]
Since  `embodied/` is moved outside dreamerv3, but now it makes pip installation not work. Can we either move it back into dreamerv3 sub directory? Or did you intend to have two packages? If so I guess there should be two setup.pys and a different directory structure.

**Comment by danijar** [comment]
Good point. I'm not maintaining the pip package right now, but ultimately I think having separate `embodied` and agent packages makes sense.
